app/code/Magento/QuoteGraphQl/Model/Resolver/AddSimpleProductsToCart.php

```diff
@@ -79,7 +79,12 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
         $currentUserId = $context->getUserId();
         $cart = $this->getCartForUser->execute((string)$cartHash, $currentUserId);
 
-        $this->addProductsToCart->execute($cart, $cartItems);
+        try {
+            $this->addProductsToCart->execute($cart, $cartItems);
+        } catch (\Exception $exception) {
+            throw new GraphQlInputException(__($exception->getMessage()));
```

Comment:
@XxXgeoXxX Could you cover this case with API-functional tests?
Thanks

Reply:
API-functional test was added. Please, check and review